convex/accessControl.ts

```
@@ -0,0 +1,565 @@
+    switch (resourceType) {
+      case "notebook":
+        // Revoke all sessions for this notebook and user
+        const sessions = await ctx.db
+          .query("notebookSessions")
+          .filter((q) => 
+            q.and(
+              q.eq(q.field("notebookId"), args.resourceId as Id<"notebooks">),
+              q.eq(q.field("userId"), args.userId as Id<"users">)
+            )
+          )
+          .collect();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)